models/UserModel.ts

```diff
@@ -113,6 +117,7 @@ export class UserModel extends BaseEntity {
       major: this.major,
       bio: this.bio,
       points: this.points,
+      userSocialMedia: this.userSocialMedia,
     };
```

Comment:
This field here should be optional, since the join of Users <-> UserSocialMedia might not always happen (since the relationship is not eager - we're only joining on `getPublicProfile()` or `getFullUserProfile()`.

I left a similar comment on the Resumes PR which explains this further and how to make this field optional: https://github.com/acmucsd/membership-portal/pull/319#discussion_r1023404624